docs/core/deploying/index.md

```diff
@@ -1,95 +1,143 @@
+| [runtime-dependent executable](#publish-runtime-dependent)            | ✔️       | ✔️       |         | [`dotnet publish`](../tools/dotnet-publish.md) |
+| [runtime-dependent executable](#publish-runtime-dependent)            | ✔️       | ✔️       |         | [`dotnet publish -r <RID> --self-contained false`](../tools/dotnet-publish.md) |
+| [runtime-dependent cross-platform binary](#publish-runtime-dependent) | ✔️       | ✔️       | ✔️       | [`dotnet publish`](../tools/dotnet-publish.md) |
+| [self-contained executable](#publish-self-contained)                  | ✔️       | ✔️       | ✔️       | [`dotnet publish -r <RID>`](../tools/dotnet-publish.md) |
 
-- Framework-dependent executables. Produces an executable that runs on a target platform. Similar to FDDs, framework-dependent executables (FDE) are platform-specific and aren't self-contained. These deployments still rely on the presence of a shared system-wide version of .NET Core to run. Unlike an SCD, your app only contains your code and any third-party dependencies that are outside of the .NET Core libraries. FDEs produce an executable that runs on the target platform.
+For more information about the `dotnet publish` command, see [.NET Core dotnet publish command](../tools/dotnet-publish.md).
 
-## Framework-dependent deployments (FDD)
+## Produce an executable
 
-For an FDD, you deploy only your app and third-party dependencies. Your app will use the version of .NET Core that's present on the target system. This is the default deployment model for .NET Core and ASP.NET Core apps that target .NET Core.
+Executables are created per-operating system, per-CPU architecture, and aren't cross-platform. When publishing your app and creating an executable, you can publish the app as [self-contained](#publish-self-contained) or [runtime-dependent](#publish-runtime-dependent). Publishing your app self-contained includes the .NET Core runtime with your app, and users of your app don't have to worry about installing .NET Core before running your app. Apps published runtime-dependent don't include the .NET Core runtime and libraries; only your app and 3rd-party dependencies are included.
 
-### Why create a framework-dependent deployment?
+The following commands produce an executable:
 
-Deploying an FDD has a number of advantages:
+| Type                                                                  | SDK 3.1 | SDK 3.0 | SDK 2.1 | Command | 
+| --------------------------------------------------------------------- | --------| ------- | ------- | ------- |
+| [runtime-dependent executable](#publish-runtime-dependent)            | ✔️       | ✔️       |         | [`dotnet publish`](../tools/dotnet-publish.md) |
+| [runtime-dependent executable](#publish-runtime-dependent)            | ✔️       | ✔️       |         | [`dotnet publish -r <RID> --self-contained false`](../tools/dotnet-publish.md) |
```

Comment:
ditto, it might be helpful to differentiate these rows and collapse the 3.x columns.